frontend/src/components/pages/teacher/DistributeAssessmentPage.tsx

```diff
@@ -1,8 +1,99 @@
-import React from "react";
-import { Text } from "@chakra-ui/react";
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
This line (and the others in this PR) probably shouldn't be necessary? normally eslint ignores `React` as an extra import because it used to be a required import. (As an aside, we should update our eslint config to not require it at some point but probably later)

Reply:
ohh okay, I thought this was just for the react import since it appeared on this line. good w leaving it in for now then for any files it's needed for